src/Avalonia.Visuals/Platform/ITouchPlatformSettings.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Avalonia.Platform
+{
+    public interface ITouchPlatformSettings
+    {
+        Size TouchDoubleClickSize { get; }
```

Comment:
It would be very nice to comment interface members like this. These are critical components and everyone needs to understand their purpose easily without searching code. If I had my way no interfaces would be committed without being fully documented :)

Reply:
I had another thought here: In UWP terminology seems to be moving towards 'Pressed' or 'Tapped' instead of 'Click'. Click is related to mouse input. Tapped is related to touch input. 'Pressed' is a good term for all cases.

Reply:
Tapped is better, since event is simply called "DoubleTapped" in both UWP and Avalonia.
But I agree, click seems wrong here now.

Reply:
@Takoooooo what about moving touch settings to the general platform settings and naming it "TouchDoubleTapSize" and "TouchDoubleTapTime".

Reply:
Current naming complies with the naming used in IPlatformSettings if we are planning to rename it we will need to rename it in IPlatformSettings also,I don't see any purpose.

Reply:
I'm surprised you don't see any purpose. Good API naming is important. 

If tapped was used, touch prefix is redundant meaning "DoubleTappedSize" is best. This is especially true since you are pulling touch settings into their own interface separate from the rest.